Lesson3.ipynb

```diff
@@ -29,6 +32,440 @@
+    "\n",
+    "Here is another example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's print the first number divisible by 13 between 30 and 60 (inclusive)\n",
+    "for num in range(30, 61):  # loop through the list\n",
+    "    if num % 13 == 0:      # % is modulo, num % 13 == 0 means num divides 13 has 0 as remainder\n",
+    "        print(num)    \n",
+    "        break              # break is used to break out of the loop, we don't want to print all number greater than 50\n",
+    "                           # but only the first one, so we break out after the first time the condition is met\n",
+    "        \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num = 30\n",
+    "while num <= 60 and num % 13 != 0:   # num % 13 != 0 means num divides 13 has non-zero remainder\n",
+    "    num = num + 1\n",
+    "    \n",
+    "print(num)  # this should prove that the loop stops at the correct value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's do a practice with `while` loop:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# use while loop to print numbers divisible by 6 between 1 and 40 (each number is on one line)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Function\n",
+    "\n",
+    "### What is a function?\n",
+    "In computer programming, a function (subroutine) is a sequence of program instructions that perform a specific task, packaged as a unit. This unit can then be used in programs wherever that particular task should be performed. (Wikipedia)\n",
+    "\n",
+    "When I attending a Bluejeans meeting, here are what I usually do:\n",
+    "> Open Bluejeans\n",
+    "\n",
+    "> Enter the meeting ID - 4482147579 \n",
+    "\n",
+    "> Participate in the meeting\n",
+    "\n",
+    "> Leave the meeting at the end\n",
+    "\n",
+    "> Enter the rating\n",
+    "\n",
+    "These steps are usually similar for my other Bluejeans meetings, except the 2nd step I might enter a different meeting ID\n",
+    "\n",
+    "In coding we can define a function like this:\n",
+    "```Python\n",
+    "def join_bluejeans_meeting(meeting_id):\n",
+    "    ...                               # omitted to save space\n",
+    "    enter_meeting_id using meeting_id # making use of the meeting_id in parameter\n",
+    "    ...                               # omitted to save space\n",
+    "    \n",
+    "# if I want to attend meeting 4482147579, I can do\n",
```

Comment:
should not share your Bluejeans meeting ID?

Reply:
It's yours :P